resolversrv/tests/data/datalinks.sql

```diff
@@ -9,10 +9,10 @@ CREATE TABLE datalinks(
       );
 
 
-INSERT INTO public.datalinks VALUES ('2013MNRAS.435.1904M', 'ARTICLE',      'EPRINT_HTML', '{http://arxiv.org/abs/1307.6556}', '{}', 0);
-INSERT INTO public.datalinks VALUES ('2013MNRAS.435.1904M', 'ARTICLE',      'EPRINT_PDF',  '{http://arxiv.org/pdf/1307.6556}', '{}', 0);
-INSERT INTO public.datalinks VALUES ('2013MNRAS.435.1904M', 'ARTICLE',      'PUB_HTML',    '{http://dx.doi.org/10.1093%2Fmnras%2Fstt1379}', '{}', 0);
-INSERT INTO public.datalinks VALUES ('2013MNRAS.435.1904M', 'ARTICLE',      'PUB_PDF',     '{http://mnras.oxfordjournals.org/content/435/3/1904.full.pdf}', '{}', 0);
+INSERT INTO public.datalinks VALUES ('2013MNRAS.435.1904M', 'ESOURCE',      'EPRINT_HTML', '{http://arxiv.org/abs/1307.6556}', '{}', 0);
```

Comment:
i'd like all data to be part of the unittest - from experience, that is a good thing (because configs and detached can change separately and break the tests; or the code changes and tests break because external files were not updated)

Reply:
done.